packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -570,13 +566,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const tryFetchBlob = async <T>(blobName: string): Promise<T | undefined> => {
             const blobId = context.baseSnapshot?.blobs[blobName];
             if (context.baseSnapshot && blobId) {
-                if (context.attachState === AttachState.Attached) {
-                    // IContainerContext storage api return type still has undefined in 0.39 package version.
-                    // So once we release 0.40 container-defn package we can remove this check.
-                    assert(storage !== undefined, 0x1f5 /* "Attached state should have storage" */);
-                    return readAndParse<T>(storage, blobId);
-                }
-                return readAndParseFromBlobs<T>(context.baseSnapshot.blobs, blobId);
```

Comment:
Is this change backwards compatible? Like will new version of runtime always work with older version of loader? Since loader may not have the new code; may not have the code to work around this part.

Also will new version of loader (which does not have `readAndParseFromBlobs`) work with older version of runtime? Since old runtime will try to call `readAndParseFromBlob`? I think the function should exist for another +2 versions for back-compat.

Reply:
Yes previous versions of loader have the blobs in storage. So newer runtime will be able to fetch it.

Reply:
Hmm, I missed the second part as the function should remain as it is in loader and older runtime will call it.

Reply:
Restored.